chapters/en/chapter10/4.mdx

```diff
@@ -0,0 +1,343 @@
+
+“Consider the task of training a model on a speech recognition dataset”
+
+Whereas for spontaneous speech, we can expect a more colloquial style of speech, with the inclusion of repetitions, hesitations and false-starts:
+
+“Let’s <uhm> take a look at how <silence> you’d you’d go about training a model on <uhm> a sp- speech recognition dataset” (this is an extreme example)
+
+#### 4. Transcription style
+Whether the text has punctuation, casing or both. If we want a system to generate fully formatted text that could be used for a publication or meeting transcription, we require training data with punctuation and casing. If we just require the spoken words in an unformatted structure, neither punctuation nor casing are necessary.
+
+A summary of datasets on the HF Hub:
+
+|                | Train Hours | Domain                                | Speaking Style         | Transcription Casing | Transcription Punctuation | Recommended Usage                                 |
+|----------------|-------------|---------------------------------------|------------------------|----------------------|---------------------------|---------------------------------------------------|
+| LibriSpeech    | 960         | Audiobooks                            | Narrated               | ❌                    | ❌                         | Academic benchmarks                               |
+| Common Voice 9 | 2224        | Wikipedia text + crowd-sourced speech | Narrated               | ✅                    | ✅                         | Day-to-day speech                                 |
+| TEDLIUM        | 452         | TED talks                             | Narrated               | ❌                    | ❌                         | Technical scientific, political and social topics |
+| Voxpopuli      | 543         | European Parliament recordings        | Spontaneous            | ❌                    | ❌                         | Non-native English speakers                       |
+| GigaSpeech     | 10000       | Audiobook, podcast, youtube           | Narrated & spontaneous | ❌                    | ✅                         | Robustness over multiple domains                  |
+| SPGISpeech     | 5000        | Financial meetings                    | Narrated & spontaneous | ✅                    | ✅                         | Fully formatted transcriptions                    |
+
+
+Let’s take an example where we want to train a speech recognition system to transcribe speech on the topic of machine learning (highly relevant!). Our speech is going to be narrated (scripted), and we’re not worried about punctuation or casing on our text output. From our reference table, it looks like TED-LIUM is a good choice of dataset!
+
+[//]: # On review, this sub-section might be too involved. Happy to omit it and dive straight into how to load a dataset.
```

Comment:
Personally, I think the content is excellent and informative - we are not following the same format as the `transformers` tutorials (which tend to be much more to the point / focus on the code)

Reply:
Okay brill! Retaining it in that case 🙂